app/src/main/res/values/strings.xml

```diff
@@ -691,4 +691,5 @@ Upload your first media by tapping on the add button.</string>
   <string name="feedback_sharing_data_alert">Please remove from this email any information that you are not comfortable sharing publicly. Also, please be aware that your email address with which you are posting, and the associated name and profile picture, will be visible publicly.</string>
   <string name="achievements_unavailable_beta">Achievements are only available in the prod flavor.</string>
   <string name="leaderboard_unavailable_beta">The leaderboard is only available in the prod flavor.</string>
+  <string name="select_one_reason">Please, Select At-least one reason</string>
```

Comment:
`Please select at least one reason.`

Also the name of the string could be "no_reason_selected"